src/braket/device_schema/jaqcd_device_action_properties.py

```diff
@@ -75,3 +77,4 @@ class JaqcdDeviceActionProperties(DeviceActionProperties):
 
     supportedOperations: List[str]
     supportedResultTypes: Optional[List[ResultType]]
+    explicitQubits: bool = False
```

Comment:
Have you tested this with the SDK and AwsDevice?

Reply:
Yep